### ISSUE TITLE
Revisit presentation of titles in Voice

### ISSUE THREAD

**Issue by sabbamitta** [sc-voice/sc-voice#486]
#### Description

The presentation of titles in Voice is somewhat inconsistent and confusing. It should be revisited after current structure changes of SC have settled, and all relevant "names" files have been translated into German.

See discussion here: https://discourse.suttacentral.net/t/sc-voice-development-cycle-eva/18146/179?u=sabbamitta
